instrumentation/grape/lib/opentelemetry/instrumentation/grape/event_handler.rb

```diff
@@ -4,6 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+require 'rack'
```

Comment:
hmm interesting. I think this needs to be a test dependency and not a production dependency if that makes sense.

If the app does not use Rack, then it should not load this file. 

That _should_ be guarded by this:

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb#L22

Grape requires rack as a Runtime dependency so this should not be necessary: https://github.com/ruby-grape/grape/blob/master/grape.gemspec#L27

I am confused why the test suite is not requiring `rack` since it is loading `grape`.

Reply:
You're correct, I believe this is a redundant require statement. Let me create a follow-up PR to remove this. We can also safely not add rack as a development dependency

Reply:
as promised the followup PR https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/450